projects/components/src/table/renderers/data-cell/table-data-cell-renderer.component.ts

```diff
@@ -58,7 +99,14 @@ export class TableDataCellRendererComponent implements OnInit {
       this.injector
     );
 
+    // Use cell renderer to initialize value for filtering
+    this.componentRef = this.componentFactoryResolver
```

Comment:
Oh god - this is why you added initialize. We're instantiating a component but never attaching it to a template and just manipulating it to pull value out of it... we gotta come up with a better way of doing this.

Reply:
https://github.com/hypertrace/hypertrace-core-ui/pull/75/files#r469530453